scripts.py

```diff
@@ -1269,77 +1267,114 @@ def make_query(self, _db, identifier_type, identifiers, log=None):
                     'Restricted to identifier type "%s".' % identifier_type
                 )
             query = query.filter(Identifier.type==identifier_type)
-        '''
-        'SELECT editions.id AS editions_id, editions.data_source_id AS editions_data_source_id, editions.primary_identifier_id AS editions_primary_identifier_id, 
-        editions.title AS editions_title, editions.sort_title AS editions_sort_title, editions.subtitle AS editions_subtitle, editions.series AS editions_series, 
-        editions.series_position AS editions_series_position, editions.permanent_work_id AS editions_permanent_work_id, editions.author AS editions_author, 
-        editions.sort_author AS editions_sort_author, editions.language AS editions_language, editions.publisher AS editions_publisher, 
-        editions.imprint AS editions_imprint, editions.issued AS editions_issued, editions.published AS editions_published, editions.medium AS editions_medium, 
-        editions.cover_id AS editions_cover_id, editions.cover_full_url AS editions_cover_full_url, editions.cover_thumbnail_url AS editions_cover_thumbnail_url, 
-        editions.open_access_download_url AS editions_open_access_download_url, editions.simple_opds_entry AS editions_simple_opds_entry, 
-        editions.extra AS editions_extra \n
-        FROM editions, identifiers \n
-        WHERE editions.primary_identifier_id IN (:primary_identifier_id_1) AND identifiers.type = :type_1'
-        '''
 
         if log:
             log.info(
                 "Processing %d editions.", query.count()
             )
+            print "Processing %d editions.", query.count()
+
         return query.order_by(Edition.id)
 
 
     def run(self, batch_size=10):
         param_args = self.parse_command_line(self._db)
         
-        #if param_args.identifiers:
+        if param_args.identifiers:
             # we're asked about a specific set of work contributors
-            #identifier_ids = [x.id for x in param_args.identifiers]
+            identifier_ids = [x.id for x in param_args.identifiers]
 
         self.query = self.make_query(
             self._db, param_args.identifier_type, param_args.identifiers, self.log
         )
 
         editions = True
         offset = 0
+        output = "ContributorID|\tSortName|\tDisplayName|\tComputedSortName|\tMatchRatio_Etc|\tResolution"
+        print output.encode("utf8")
```

Comment:
Good to know.  I'll write the class reference into my personal notes, and let me know if you want me to change the code.